docs/src/main/asciidoc/secretmanager.adoc

```diff
@@ -80,12 +99,14 @@ The auto-configured `SecretManagerTemplate` bean can be customized using configu
 By default, Spring Cloud GCP Secret Manager will authenticate using Application Default Credentials.
 This can be overridden using the authentication properties.
 
+NOTE: All of the below settings must be specified in a https://cloud.spring.io/spring-cloud-commons/multi/multi__spring_cloud_context_application_context_services.html#_the_bootstrap_application_context[`bootstrap.properties`] (or `bootstrap.yaml`) file which is the properties file used to configure settings for bootstrap-phase Spring configuration.
+
 |===
 | Name | Description | Required | Default value
```

Comment:
Since now there is a single table of properties, we should move it up to be right under the dependency section, just like other chapters we have.

Reply:
Done.